pkgs/games/steam/fhsenv.nix

```diff
@@ -68,13 +56,6 @@ let
 
   runSh = writeScript "run.sh" ''
```

Comment:
Since this variable is unused, this is redundant, no?

Reply:
it is, thanks